src/metatrain/utils/scaler.py

```diff
@@ -0,0 +1,237 @@
+
+        return self
+
+    def forward(
+        self,
+        outputs: Dict[str, TensorMap],
+    ) -> Dict[str, TensorMap]:
+        """
+        Scales all the targets in the outputs dictionary back to their
+        original scale.
+
+        :param outputs: A dictionary of target quantities and their values
+            to be scaled.
+
+        :raises ValueError: If an output does not have a corresponding
+            scale in the scaler model.
+        """
+
+        for output_name in outputs:
+            if output_name.startswith("mtt::aux::") or output_name == "features":
+                continue
+            if output_name not in self.outputs.keys():
+                raise ValueError(
+                    f"output key {output_name} is not supported by this scaler "
+                    "model."
+                )
+
+        scaled_outputs: Dict[str, TensorMap] = {}
+        for target_key, target in outputs.items():
+            if target_key.startswith("mtt::aux::") or target_key == "features":
+                scaled_outputs[target_key] = target
+            else:
+                scale = float(
+                    self.scales[self.output_name_to_output_index[target_key]].item()
+                )
+                scaled_target = metatensor.torch.multiply(target, scale)
+                scaled_outputs[target_key] = scaled_target
+
+        return scaled_outputs
+
+    def _add_output(self, target_name: str, target_info: TargetInfo) -> None:
+
+        self.outputs[target_name] = ModelOutput(
+            quantity=target_info.quantity,
+            unit=target_info.unit,
+            per_atom=True,
+        )
+
+        self.scales = torch.cat(
+            [self.scales, torch.tensor([1.0], dtype=self.scales.dtype)]
+        )
+        self.output_name_to_output_index[target_name] = len(self.scales) - 1
+
+    def get_scales_dict(self) -> Dict[str, float]:
+        """
+        Return a dictionary with the scales for each output and output gradient.
+
+        :return: A dictionary with the scales for each output and output gradient.
+            These correspond to the standard deviation of the targets in the
+            original dataset. The scales for each output gradient are the same
+            as the corresponding output.
+        """
+
+        scales_dict = {
+            output_name: self.scales[output_index].item()
+            for output_name, output_index in self.output_name_to_output_index.items()
+        }
+        # Add gradients if present. They have the same scale as the corresponding output
+        for output_name in list(scales_dict.keys()):
+            gradient_names_for_output = self.dataset_info.targets[output_name].gradients
+            for gradient_name in gradient_names_for_output:
+                scales_dict[output_name + "_" + gradient_name + "_gradients"] = (
+                    scales_dict[output_name]
+                )
+        return scales_dict
+
+
+def remove_scale(
```

Comment:
Other names suggestions:

```suggestion
def scale_to_one(
```

```suggestion
def normalize_scale(
```

This is the opposite of `Scaler.forward`, right?

Reply:
I think `remove_scale` is a good name because it reminds of `remove_additive` that can be used with additive models

Reply:
I don't agree: `remove` makes sense for something you substract from the target, but less so here. It is not very important though.

Reply:
Ok, opening issue